js/interface.js

```diff
@@ -2611,11 +2667,33 @@ var DynamicLists = (function() {
           item.value = item.fieldValue;
         }
 
+
+        item.filterModifier = {
+          default: $('#date-field-' + item.id).val()
+        };
+
+        item.modifierValue = {
+          default: $('#number-field-' + item.id).val()
+        };
+
         if (item.logic === 'empty' || item.logic === 'notempty') {
-          item.valueType = null;
+          item.valueType = '';
           item.value = '';
         }
 
+        if (item.logic === 'datebetween') {
+          var valueDateFrom = $('#date-from-' + item.id).val();
+          var valueDateTo = $('#date-to-' + item.id).val();
+          var valueNumberFrom = $('#number-field-from-' + item.id).val();
+          var valueNumberTo = $('#number-field-to-' + item.id).val();
+
+          item.filterModifier.to = valueDateTo;
```

Comment:
When the logic is `between`, we use the filter `item.value = { from: valueFrom, to: valueTo }`.

I think we can do the same here with `databetween`:

```js
ite.value = {
  from: { value: 'today' }, // If there's no offset required, don't add the offset property
  to: { value: 'todayadddays', offset: 2 } // Parse the offset value into a number as specified
}
```